modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/aquatic.dm

```diff
@@ -80,3 +80,9 @@
 	if(BMS)
 		markings = assemble_body_markings_from_set(BMS, passed_features, src)
 	return markings
+
+/datum/species/aquatic/get_species_description()
+	return "Placeholder Description - Fill me!"
+
+/datum/species/aquatic/get_species_lore()
+	return list("Placeholder Lore - Fill me!")
```

Comment:
You forgot to use the vars you made, here.

Reply:
Woops!